commitizen/commands/init.py

```diff
@@ -109,17 +120,37 @@ def _ask_tag_format(self, latest_tag) -> str:
                 tag_format = "$version"
         return tag_format
 
-    def _install_pre_commit_hook(self):
+    def _search_pre_commit(self):
+        return shutil.which("pre-commit") is not None
+
+    def _exec_install_pre_commit_hook(self, hook_types: List[str]):
+        if not hook_types:
+            raise ValueError("At least 1 hook type should be provided.")
+        cmd_str = "pre-commit install " + "".join(
+            f"--hook-type {ty}" for ty in hook_types
+        )
+        c = cmd.run(cmd_str)
+        if c.return_code != 0:
+            out.error(f"Error running {cmd_str}. Outputs are attached below:")
```

Comment:
nitpick: I might suggest invoke one `out.error` and concatenation the string instead.